Modules/Scripted/EditorLib/HelperBox.py

```diff
@@ -584,8 +582,6 @@ def updateStructures(self,caller=None, event=None):
     self.setMergeButton.setDisabled(not self.master)
 
     # reset to a fresh model
-    self.brushes = []
-    self.items = []
```

Comment:
The reason for these lists was to handle memory management -- the QItems are not QObjects so they fall into a gray zone as far as PythonQt.

http://massmail.spl.harvard.edu/public-archives/slicer-devel/2014/016179.html

Has this been fixed?